src/server_manager/install_scripts/install_server.sh

```diff
@@ -102,7 +104,7 @@ function confirm() {
   echo -n "> $1 [Y/n] "
   local RESPONSE
   read RESPONSE
-  RESPONSE=$(echo "$RESPONSE" | tr '[A-Z]' '[a-z]')
+  RESPONSE="${RESPONSE,,}" # Convert to lowercase
```

Comment:
Wat??? What's going on there?
I think the previous version was more readable. It's hard to find documentation to explain that.

Reply:
OK, reverted.  (This formula was suggested [here](https://stackoverflow.com/questions/2264428/how-to-convert-a-string-to-lower-case-in-bash), using a special case of [parameter expansion](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion).)

Reply:
BTW, the trick seems to be for bash 4 only. It doesn't work on my machine, and perhaps not on many images.